src/leveldb/util/env_win.cc

```diff
@@ -442,202 +423,63 @@ void Win32RandomAccessFile::_CleanUp()
     }
 }
 
-size_t Win32MapFile::_Roundup( size_t x, size_t y )
+Win32WritableFile::Win32WritableFile(const std::string& fname)
+    : filename_(fname)
 {
-    return ((x + y - 1) / y) * y;
+    std::wstring path;
+    ToWidePath(fname, path);
+    DWORD Flag = PathFileExistsW(path.c_str()) ? OPEN_EXISTING : CREATE_ALWAYS;
```

Comment:
The goal is to have `f=fopen(filename, "wb")` semantics. It's possible that this could be improved, but indeed I just copied the code from the previous WritableFile implementation.